src/WP_CLI/CommandWithUpgrade.php

```diff
@@ -2,6 +2,7 @@
 
 namespace WP_CLI;
 
+use Composer\Package\Version\VersionParser;
```

Comment:
Isn't it `Composer\Semver\VersionParser`? Or is that something else? 🤔

Reply:
```suggestion
use Composer\Semver\VersionParser;
```

Looks like `Composer\Package\Version\VersionParser` simply subclasses `Composer\Semver\VersionParser`.

We can use the parent class though